gtk4/src/auto/widget.rs

```diff
@@ -59,6 +59,13 @@ impl Widget {
     }
 }
 
+impl fmt::Display for Widget {
```

Comment:
Is this "automatically" generated code? if not, it should be moved to src/widget.rs

Reply:
This is generated.

Reply:
It weird how gir moved it to the top of the file instead of the bottom, which confused me :)